udf-compiler/src/main/scala/com/nvidia/spark/udf/CFG.scala

```diff
@@ -0,0 +1,328 @@
+          val lowOffset = defaultOffset + 4
+          val low = codeIterator.s32bitAt(lowOffset)
+          val highOffset = lowOffset + 4
+          val high = codeIterator.s32bitAt(highOffset)
+          val tableOffset = highOffset + 4
+          val table = List.tabulate(high - low + 1) { i =>
+            (low + i, offset + codeIterator.s32bitAt(tableOffset + i * 4))
+          } :+ default
+          collectLabelsAndEdges(
+            codeIterator, constPool,
+            labels ++ table.unzip._2,
+            edges + (offset -> table))
+        case Opcode.LOOKUPSWITCH =>
+          val defaultOffset = (offset + 4) / 4 * 4
+          val default = (-1, offset + codeIterator.s32bitAt(defaultOffset))
+          val npairsOffset = defaultOffset + 4
+          val npairs = codeIterator.s32bitAt(npairsOffset)
+          val tableOffset = npairsOffset + 4
+          val table = List.tabulate(npairs) { i =>
+            (codeIterator.s32bitAt(tableOffset + i * 8),
+             offset + codeIterator.s32bitAt(tableOffset + i * 8 + 4))
+          } :+ default
+          collectLabelsAndEdges(
+            codeIterator, constPool,
+            labels ++ table.unzip._2,
+            edges + (offset -> table))
+        case Opcode.GOTO | Opcode.GOTO_W =>
+          // goto statements have a single address target, we must go there
+          val getOffset = if (opcode == Opcode.GOTO) {
+                            codeIterator.s16bitAt(_)
+                          } else {
+                            codeIterator.s32bitAt(_)
+                          }
+          val labelOffset = offset + getOffset(offset + 1)
+          collectLabelsAndEdges(
+            codeIterator, constPool,
+            labels + labelOffset,
+            edges + (offset -> List((0, labelOffset))))
+        case Opcode.IF_ACMPEQ | Opcode.IF_ACMPNE |
+             Opcode.JSR | Opcode.JSR_W | Opcode.RET =>
+          val instructionStr = InstructionPrinter.instructionString(codeIterator, offset, constPool)
+          throw new SparkException("Unsupported instruction: " + instructionStr)
+        case _ => collectLabelsAndEdges(codeIterator, constPool, labels, edges)
+      }
+    } else {
+      // base case
+      (labels, edges)
+    }
+  }
+
+  @tailrec
+  private def createInstructionTable(codeIterator: CodeIterator, constPool: ConstPool,
+                                     instructionTable: SortedMap[Int, Instruction] = SortedMap())
+  : SortedMap[Int, Instruction] = {
+    if (codeIterator.hasNext) {
+      val offset = codeIterator.next
+      val instructionStr = InstructionPrinter.instructionString(codeIterator, offset, constPool)
+      val instruction = Instruction(codeIterator, offset, instructionStr)
+      createInstructionTable(codeIterator, constPool,
+        instructionTable + (offset -> instruction))
+    } else {
+      instructionTable
+    }
+  }
+
+  @tailrec
+  private def createBasicBlocks(labels: SortedSet[Int],
```

Comment:
some of the styling in the code has changed, please add the codestyle xml to your IntelliJ and refactor the code:

https://github.com/NVIDIA/spark-rapids/blob/branch-0.2/docs/dev/idea-code-style-settings.xml

Let us know if you run into issues here.

Reply:
Also, an `mvn verify` should output scalastyle errors.

Reply:
Updated.